### PR TITLE
Environment details style improvements

### PR DESCRIPTION
**Pull request by clarkepaul**
Basic style additions, Added popup for repo url

### CODE REVIEW

File: templates/Layout/DNRoot_environment.ss [hunk 1]
```diff
@@ -37,49 +38,64 @@
...
-							<a href="$CurrentBuild.Link">View deployment log</a>
-						</li>
-					<% else %>
-						<li>
-							No deployments have been made
-						</li>
-					<% end_if %>
-				</ul>
+							<%-- Display current build on environment --%>
+							<li>
```
Comment (hafriedlander) [inline templates/Layout/DNRoot_environment.ss]: Why did you discard my changes here?

**Comment by hafriedlander** [comment]
This breaks the project and environment pages. Closing for now so someone doesn't accidentally merge.

File: templates/Layout/DNRoot_environment.ss (hunk 1 above)
Comment (clarkepaul) [inline templates/Layout/DNRoot_environment.ss]: I thought I just moved - I did get a conflict but didn't see any changes

Comment (clarkepaul) [inline templates/Layout/DNRoot_environment.ss]: is the only difference <% include GitBuildReference %> so far that is the only thing missing right?

Comment (clarkepaul) [inline templates/Layout/DNRoot_environment.ss]: BTW $BareURL doesn't return anything so I just used $URL

**Pull request closed without merging**